detail.html

```diff
@@ -0,0 +1,40 @@
+<%inherit file="app_base.htm" />
+
+<%block name="detail">
+
+    <div class="col-8 text-center">
+        <div id="description">
+
+            <ul id="smallPics">
+            % for img in product.image_urls():
+                <% 
+                    # get the name of the pictures
+                    start  = img.rfind('/')+1
+                    finish = img.rfind('.')
+                    alt = img[start:finish]
+                %>

```

Comment:
I honestly don't know what this section above in the <% %> section does or why it is needed.

Reply:
This allows me to take an img src attribute and cut out the name of the image (e.g. /catalog/media/products/imgname.jpg becomes 'imgname'. I then use this as the alt attribute on the image.

Reply:
ite lets me add specific alt tab text (name of the picture)